### DISCUSSION TITLE
[v5] Cannot build a non-typescript project. Make sure to run "node ace build" from the project root

### DISCUSSION THREAD

**Discussion by bitkidd** [adonisjs/core#1743] [Help] [answered]
Hey!
I have just updated packages and got an error: `[ error ]  Cannot build a non-typescript project. Make sure to run "node ace build" from the project root`.
The packages installed: 
```
"devDependencies": {
    "adonis-preset-ts": "^1.0.4",
    "eslint": "^7.11.0",
    "eslint-config-airbnb": "^18.2.0",
    "eslint-config-prettier": "^6.12.0",
    "eslint-plugin-adonis": "^1.0.15",
    "eslint-plugin-import": "^2.22.1",
    "eslint-plugin-prettier": "^3.1.4",
    "pino-pretty": "^4.3.0",
    "prettier": "^2.1.2",
    "typescript": "^4.0.3",
    "youch": "^2.1.1",
    "youch-terminal": "^1.0.1"
  },
  "dependencies": {
    "@adonisjs/ace": "^7.1.4",
    "@adonisjs/assembler": "^2.1.5",
    "@adonisjs/core": "^5.0.1",
    "@adonisjs/fold": "^7.0.6",
    "@adonisjs/lucid": "^9.0.0",
    "cheerio": "^1.0.0-rc.3",
    "md5": "^2.3.0",
    "mkdirp": "^1.0.4",
    "pdf-parse": "^1.1.1",
    "pg": "^8.4.1",
    "proxy-addr": "^2.0.6",
    "reflect-metadata": "^0.1.13",
    "source-map-support": "^0.5.19"
  },
```

**Comment by thetutlage** [comment, marked as answer]
Why you want to build a non typescript project? Just run it as it is. 

As a side note: You should be using the `preview` version of the core. 

```sh
npm i @adonisjs/core@alpha
```

**Reply by bitkidd** [reply]
I was trying to deploy v5 via Cleavr. Just reinstalled package, tried to start and got a new error:
`TypeError: Cannot read property 'fatal' of undefined at /node_modules/@adonisjs/core/build/src/Ignitor/Ace/index.js:156:28`

**Reply by bitkidd** [reply]
Okay, installed an alpha version of ace and it worked. The version thing is a bit confusing by now as we have v4 and v5 at the same time.
Thank you.
